docs/basics/basic_scenario.md

```diff
@@ -0,0 +1,56 @@
+# Scenario
+
+Here is the basic scenario structure.  There is only one Scenario for one phase.  Each scenario consists of sequences.  When many sequences are specified in below synstax, they are executed sequentially.  If parallel execution are required, please see [initialSequences and sequences]({{ "/userguide/benchmark/scenario.html" | absolute_url }}).  For more detail, please see [scenario]({{ "/docs/concepts.html#scenario" | absolute_url }})
+
+Note: httpRequest is a **step**, it will be the same level as all the other STEPS.
+
+```yaml
+scenario:
+- SEQUENCE_NAME:
+  (Optional - STEPS)
+  - httpRequest: ...
+```
+
+Example: 
+
+```yaml
+scenario:
+- queryFlight:
+  - thinkTime:
+      duration: 3s
+  - set: isOneWay <- false
+  - httpRequest: ...
+```
+
+## Sequences of Scenario
+
+```yaml
+scenario:
+- SEQUENCE_NAME:
+ (Optional - STEPS)
+  - httpRequest: ...
+- SEQUENCE_NAME:
+ (Optional - STEPS)
+  - httpRequest: ...
+- SEQUENCE_NAME:
+ (Optional - STEPS)
+  - httpRequest: ...
```

Comment:
This looks as if every sequence would have to end up with a `httpRequest`. Can you think about a better example?

Reply:
oh, I have never come to my mind for not using httpRequest. let me think..

Reply:
Check out https://hyperfoil.github.io/quickstart/quickstart3.html - in the first sequence ('home') we do some requests at the beginning and then process the responses in a series of steps.